internal/driver/interactive.go

```diff
@@ -135,6 +126,23 @@ func interactive(p *profile.Profile, o *plugin.Options) error {
 	}
 }
 
+// groupOptions returns an map containing all non-empty groups
+// mapped to an array of the option names in that group in
+// sorted order.
+func groupOptions() map[string][]string {
```

Comment:
I'd make this a function on the variables type like I mentioned in the original comment about this, but this is not too important.

Reply:
Done.